classes/class-wc-connect-help-view.php

```diff
@@ -310,6 +322,50 @@ public function page() {
 			);
 		}
 
+		/**
+		 * @return array
+		 */
+		protected function get_tax_health_item() {
```

Comment:
I have manually tested all the flows here but it would be cool to have some unit tests for this method if possible.

Reply:
Yeah, the whole class doesn't have any tests 😅

Reply:
Ah, got it. I'm not sure what is the way to go then since it's not a crucial part of the system. If possible, it'd be great to add the tests but I don't want to add a lot of extra work to the PR if that will be difficult.